### PR TITLE
Added DICOM conversion script

### PR DESCRIPTION
**Pull request by bsekachev**
### Motivation and context
CVAT does not support DICOM format natively. 
One way to annotate DICOM data is to convert them to regular images first. The patch adds a script to support some kinds of DICOM files to .png

### How has this been tested?
Manual testing, converted:
- [CHAOS training set](https://zenodo.org/record/3362845) CT/MT
- Some multi-frame DICOM files from [here](https://www.aliza-dicom-viewer.com/download/datasets)

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: utils/dicom_converter/script.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    else:
+        logging.info('Number of found DICOM files: ' + str(len(dicom_files)))
+
+    pbar = tqdm(dicom_files)
+    for input_filename in pbar:
+        pbar.set_description('Conversion: ' + input_filename)
+        input_subpath = input_filename.split(os.sep)[1:-1]
+        input_basename = os.path.basename(input_filename)
+
+        output_subpath = os.path.join(*input_filename.split(os.sep)[1:-1]) if len(input_subpath) else ''
```
Comment (Marishka17) [inline utils/dicom_converter/script.py:79]: Maybe
```suggestion
        input_file_dir = os.path.dirname(input_filename)
        common_prefix = os.path.commonprefix([output_root_dir, input_file_dir])
        output_subpath = os.path.relpath(input_file_dir, common_prefix)
```

File: utils/dicom_converter/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+The conversion script was tested on CT, MT and some multi-frame DICOM data.
+DICOM files with series (multi-frame) are saved under the same name with a number postfix: 001, 002, 003, etc.
+
+# Installation
+
+```bash
+python3 -m venv .env
+. .env/bin/activate
+pip install -r requirements.txt
+python3 script.py input_data output_data
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply utils/dicom_converter/README.md:14]: Yes, agree

File: utils/dicom_converter/script.py (hunk 1 above)
Comment (bsekachev) [reply utils/dicom_converter/script.py]: This code was rewritten a bit

File: utils/dicom_converter/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+# Installation
+
+```bash
+python3 -m venv .env
+. .env/bin/activate
+pip install -r requirements.txt
+```
+
+# Running
```
Comment (nmanovic) [inline utils/dicom_converter/README.md:16]: Better to provide an example for a real dataset.

**Review by nmanovic — APPROVED** [review]

**Pull request merged**